components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -145,6 +145,14 @@ export class ReportingComponent implements OnInit, OnDestroy {
     }
   ];
 
+  private noValuesControlTagFilter = {
+    id: '',
+    score: 1,
+    text: 'no value',
```

Comment:
Suggest using a named constant, especially since 'no value' is being introduced across a couple files.